### ISSUE TITLE
Ubuntu 20.04 LTS (Focal Fossa) - ' ELF section in vmlinux 

### ISSUE THREAD

**Issue by oleksis** [oleksis/wsl2-linux-kernel-config#4]
compiling linux kernel sources error 255 : 

```bash
libbpf: failed to find '.BTF' ELF section in vmlinux 
FAILED: load BTF from vmlinux: No data available
```

**Comment by oleksis** [comment]
The error message `libbpf: failed to find '.BTF' ELF section in vmlinux` indicates that the `pahole` tool was unable to find the `.BTF` (BPF Type Format) section in the `vmlinux` file when generating BTF debug information.

This error can occur if the `pahole` tool is not installed or is not up-to-date. The `pahole` tool is part of the `dwarves` package and can be installed using your distribution's package manager. For example, on Ubuntu or Debian-based systems, you can install the `dwarves` package by running:

```
sudo apt-get install dwarves
```

If you already have the `dwarves` package installed, make sure it is up-to-date by running your distribution's package update command.

Another possible cause of this error is that BTF debug information is not enabled in your kernel configuration. To enable BTF debug information, you need to enable the `CONFIG_DEBUG_INFO_BTF` kernel configuration option. You can do this by editing the kernel configuration file (`.config`) directly or by using a kernel configuration tool such as `menuconfig`.

**Comment by oleksis** [comment]
Fixed updating Ubuntu  version  >=  **22.04 LTS (Jammy Jellyfish)**
